src/stats/common.ts

```diff
@@ -67,13 +67,22 @@ export interface ActionCountsType extends PlayerIndexedType {
   rollCount: number;
 }
 
+export interface InputCountsType {
+  buttons: number;
+  triggers: number;
+  joystick: number;
+  cstick: number;
+  total: number;
+}
+
 export interface OverallType extends PlayerIndexedType {
-  inputCount: number;
+  inputCounts: InputCountsType;
```

Comment:
Should this remain being called `inputCount` over `inputCounts`? I feel like a lot of places in the repo it uses `...Count` over `...Counts`.

Reply:
Oh wait I just realised, it's no longer `count` because there are now multiple `count`s...